app/test_main.py

```diff
@@ -1,3 +1,53 @@
-from app.main import get_human_age
+import pytest
+from app.main import get_human_age, convert_to_human
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "animal_age,first_year,second_year,each_year,human_age", [
+        (0, 10, 5, 1, 0),
+        (20, 1, 1, 1, 20),
+        (123, 15, 9, 5, 21),
+        (-1, 15, 9, 5, 0)
+    ]
+)
+def test_if_convert_to_human_returns_correct_value(
+    animal_age: int,
+    first_year: int,
+    second_year: int,
+    each_year: int,
+    human_age: int
+) -> None:
+    assert (convert_to_human(
+        animal_age, first_year, second_year, each_year) == human_age
+    ), (f"Animal age({animal_age})"
+        f" in human years should be equal to {human_age}")
+
```

Comment:
Write tests only for 

![image](https://user-images.githubusercontent.com/104218550/220911336-9ddb945d-4515-4db7-a86c-08470f9cd0d9.png)

These checks are redundant